main/lxcmd.c

```diff
@@ -307,7 +307,7 @@ static boolean loadPathKinds  (xcmdPath *const path, const langType language)
 			if (status > 0 
 			    && WIFEXITED (status) 
 			    && (WEXITSTATUS (status) == XCMD_NOT_AVAILABLE_STATUS))
-				notice ("xcmd recognizes %s is not available", argv[0]);
+				verbose ("xcmd recognizes %s is not available\n", argv[0]);
```

Comment:
Thank you. For a while I think this sentence is something wrong.
How about:

                  %s reports the backend driver is not avaiable.

?